components/ui-shell/index.scss

```diff
@@ -0,0 +1,243 @@
+@import "@carbon/type/scss/type";
+@import "@carbon/colors/scss/colors";
+
+// These are needed to extend default carbon component styles
+@import "carbon-web-components/scss/components/ui-shell/header";
+@import "carbon-web-components/scss/components/ui-shell/side-nav";
+
+// These are needed to apply theming
+@import "carbon-components/scss/globals/scss/css--helpers";
+@import "carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/themes/mixins";
+
+$q: "qiskit";
+$prefix: "bx";
+
+:host {
+  --header-height: 3.5rem;
+
+  @include carbon--theme($carbon--theme--white, true);
+
+  font-family: carbon--font-family("sans");
+}
+
+.#{$q}-header {
+  display: flex;
+  justify-content: space-between;
+  height: var(--header-height);
+  border-color: var(--cds-border-subtle);
+  background-color: var(--cds-ui-background);
+  color: var(--cds-text-secondary);
+
+  /* stylelint-disable-next-line selector-class-pattern */
+  svg.menu__account__icon {
+    display: block;
+    overflow: hidden;
+    box-sizing: border-box;
+    width: $spacing-07;
+    height: $spacing-07;
+    padding-top: calc($spacing-01 / 2);
+    border: $spacing-01 solid $carbon--white-0;
```

Comment:
The `spacing-*` variables are only needed to define the space between elements. For widths, heights and borders, [rem](https://de.wikipedia.org/wiki/R.E.M.)s should be fine.